### PR TITLE
fix: (TNLT-994) play icon showing underneath image

### PR DESCRIPTION
**Pull request by kamiranoff**
- https://nidigitalsolutions.jira.com/browse/TNLT-994
- fix play icon occasionally misplace in some tiles
- fix icon misplaced on orientation change on tablets

I have tested on all iPad sizes available and seems to work fine. I just uploaded the IPad 11" version of the screenshots
before
TNLT-994 - before fix

after:
Screenshot_1611238629
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-01-22 at 00 50 54
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-01-22 at 00 51 00

More screenshots on tablets:
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-01-26 at 15 06 30
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-01-26 at 15 06 05
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-01-26 at 15 05 57
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-01-26 at 15 06 20
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-01-26 at 15 06 17
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-01-26 at 15 06 33

Android:
Screenshot_1611747222
Screenshot_1611747216

### CODE REVIEW

File: packages/edition-slices/edition-front.showcase.js [hunk 1]
```diff
@@ -50,16 +50,31 @@ const sliceStories = [
     name: "Front Lead Two",
     Slice: LeadTwoFrontSlice,
   },
+  {
+    mock: mockLeadTwoFrontSlice(true),
```
Comment (c-j-j) [inline packages/edition-slices/edition-front.showcase.js:54]: Minor point, but passing an object with a name, e.g. `{showVideo: true}`, would make this easier to extend if we wanted to test other variations of a slice, and reads a bit better too IMO.

File: packages/front-page/__tests__/front-tile-summary.test.js [hunk 2]
```diff
@@ -17,6 +17,8 @@ jest.mock("@times-components-native/front-page/front-page-byline", () => ({
   FrontPageByline: "FrontPageByline",
 }));
 
+jest.mock("react-native-image-zoom-viewer", () => "ImageZoomView");
```
Comment (c-j-j) [inline packages/front-page/__tests__/front-tile-summary.test.js:20]: Would it make sense for this to go in `test-setup/setup-jest.js`?

File: packages/fixture-generator/src/mock-tile.ts [hunk 3]
```diff
@@ -4,8 +4,8 @@ import MockArticle from "./mock-article";
 class MockTile {
   tile: Tile;
 
-  constructor() {
-    const article = new MockArticle().get();
+  constructor(hasVideo: boolean = false) {
```
Comment (c-j-j) [inline packages/fixture-generator/src/mock-tile.ts:7]: similar to above comment, I think passing an object here (and to MockArticle) will help when more variations are added for tests

File: packages/responsive/src/calculateResponsiveContext.ts [hunk 4]
```diff
@@ -20,5 +21,5 @@ export const calculateResponsiveContext = (
     width >= tabletWidth,
   windowWidth: width,
   windowHeight: height,
-  orientation: height > width ? "portrait" : "landscape",
+  orientation: height > width ? Orientation.PORTRAIT : Orientation.LANDSCAPE,
```
Comment (c-j-j) [inline packages/responsive/src/calculateResponsiveContext.ts:24]: Nice enum!

File: packages/edition-slices/edition-front.showcase.js (hunk 1 above)
Comment (kamiranoff) [reply packages/edition-slices/edition-front.showcase.js:54]: good point.

File: packages/front-page/__tests__/front-tile-summary.test.js (hunk 2 above)
Comment (kamiranoff) [reply packages/front-page/__tests__/front-tile-summary.test.js:20]: definitely. will try!

**Review by c-j-j — APPROVED** [review]
Overall looks good to me 👍

File: packages/responsive/src/calculateResponsiveContext.ts (hunk 4 above)
Comment (aronshamash) [reply packages/responsive/src/calculateResponsiveContext.ts:24]: do you think some booleans would be good to have exposed here too `isPortrait` `isLandscape` so we don't have do comparisons in each component? @c-j-j @kamiranoff

File: packages/responsive/src/context.ts [hunk 5]
```diff
@@ -2,14 +2,19 @@ import { createContext } from "react";
 import { getDimensions } from "@times-components-native/utils";
 import { calculateResponsiveContext } from "./calculateResponsiveContext";
 
+export enum Orientation {
```
Comment (aronshamash) [inline packages/responsive/src/context.ts:5]: 👍

**Review by aronshamash — APPROVED** [review]
looks very good

File: packages/responsive/src/calculateResponsiveContext.ts (hunk 4 above)
Comment (c-j-j) [reply packages/responsive/src/calculateResponsiveContext.ts:24]: Yep that'll clean things up

Comment (kamiranoff) [reply packages/responsive/src/calculateResponsiveContext.ts:24]: done!

**Pull request merged**